### ISSUE TITLE
Set a limit of 500 articles to fetch, but only 50 are downloaded

### ISSUE THREAD

**Issue by ddeimeke** [wallabag/android-app#220]
I configured a limit of 500 articles to fetch from my Wallabag 2.0.2 installation. But the limit is not respected. Currently I have about 220 articles and only 50 are fetched.

**Comment by Strubbl** [comment]
Hi Dirk, 
the limitation of 50 items is done in your wallabag server instance (Configuration->RSS->number of items in feed). The setting in the app currently only affects the amount of items being displayed in the lists of the app. If you are increasing the "number of items in feed" in your wallabag server instance AND in your app, you are going to see more articles. 
This was not a problem with wallabag v1 since per default all items were content of the RSS feed.

@tcitworld I do not know, if we want this as feature in our app in some way. I mean, that if we change the number of items in our app, do we want to change this settings in our wallabag server, too? Or do we only name the setting in our app more precise? E.g. with a note to the user if the app nummer is less than the server number?

**Comment by ddeimeke** [comment]
First and foremost: Thank you!

Thinking about it. Setting the feed limit should not affect the number of items seen in the app and vice versa.

The easiest in my opinion would be

https://wallabag.deimeke.net/dirk/TOKEN/starred.xml 

contains the number of items configured in the web applications and

https://wallabag.deimeke.net/dirk/TOKEN/starred.xml?count=500

contains the number of items configured in the Android application.

Thought about adding a separate feed, but I do not think this makes any sense.

If it stays like it is, the Android App should mention that the number of items displayed is different from the number of items downloaded.

**Comment by nicosomb** [comment]
Is this issue a duplicate of #199?

**Comment by Strubbl** [comment]
Yes, I think so. It is the same issue, it is just the other way round.
